modules/nf-core/blast/blastdbcmd/meta.yml

```diff
@@ -0,0 +1,53 @@
+name: blast_blastdbcmd
+description: Retrieve entries from a BLAST database
+keywords:
+  - fasta
+  - blast
+  - database
```

Comment:
Maybe couple more tags describing what the module does actually with blast databases? Maybe `retrieval` or `extract`?